lib/sqs.go

```diff
@@ -131,10 +134,17 @@ func (sqsQueue *SQSQueue) SendSQSTxnMessage(mempoolTxn *MempoolTx) {
 		glog.Errorf("SendSQSTxnMessage: Error marshaling transaction JSON : %v", err)
 	}
 
+	messageAttributes := make(map[string]types.MessageAttributeValue)
+	messageAttributes["messageId"] = types.MessageAttributeValue{
+		DataType:    aws.String("String"),
+		StringValue: aws.String(uuid.NewString()),
```

Comment:
Seems unnecessary to have DataType: String when the field is called “StringValue”

Reply:
I would either remove datatype or just call the second field “value”

Reply:
This is actually needed, since DataType can also be Number, but would also use StringValue to specify. (This is aws sqs format).